esp/src/src-react/components/QuerySuperFiles.tsx

```diff
@@ -42,7 +42,7 @@ export const QuerySuperFiles: React.FunctionComponent<QuerySuperFilesProps> = ({
             File: {
                 label: nlsHPCC.File,
                 formatter: function (item, row) {
-                    return `<a href="#/files/${querySet}/${item}">${item}</a>`;
+                    return <Link href={`#/files/${item}`}>{item}</Link>;
```

Comment:
Dropping the querySet is on purpose (which seems sensible)?

Reply:
Yes, that's correct, with the querySet it wasn't a valid route.